### DISCUSSION TITLE
是否支持自定义 Shader？

### DISCUSSION THREAD

**Discussion by ligewei** [Tencent/libpag#510] [General] [answered]
除了基本的填空，是否支持自定义的 GPU Shader脚本？

**Comment by kevingpqi123** [comment]
这里需求的背景是啥，在一个渲染链中，pag 可以作为一个渲染节点，可以和其它节点很好的整合

**Comment by domchen** [comment, marked as answer]
一般是把PAG作为其中一个渲染节点跟其他的Shader节点串联，不是把Shader塞到PAG里。
